docs/upgrade-notes/upgrade-rds.md

```diff
@@ -0,0 +1,246 @@
+| executionMigrationParams.writeConcurrency | number | The maximum number of execution records to write concurrently to PostgreSQL. The higher this number, the less time it will take to migrate all of your data, but also the more load that will be put on your PostgreSQL database. | 10
+| executionMigrationParams.loggingInterval | number | How many records to migrate before printing a log message on the status of the migration. | 100
+| granuleMigrationParams | Object | Options for the granules data migration | `{}`
+| granuleMigrationParams.collectionId | string | A collection ID (e.g. `collection___1`) from granule DynamoDB records. If a `collectionId` is provided, then only granules for that collection will be migrated | none
+| granuleMigrationParams.granuleId | string | A specific granule ID from a DynamoDB record to select for migration. If a `granuleId` and `collectionId` are provided, the `collectionId` will be ignored. | none
+| granuleMigrationParams.parallelScanSegments | number | The number of [parallel scan] segments to use for granules data migration. The higher this number, the less time it will take to migrate all of your data, but also the more load that will be put on your PostgreSQL database. | 20
+| granuleMigrationParams.parallelScanLimit | number | The maximum number of records to return per each [parallel scan] of a segment. This option was mostly provided for testing and it is not recommended to set a value. | none
+| granuleMigrationParams.writeConcurrency | number | The maximum number of granule records to write concurrently to PostgreSQL. The higher this number, the less time it will take to migrate all of your data, but also the more load that will be put on your PostgreSQL database. | 10
+| granuleMigrationParams.loggingInterval | number | How many records to migrate before printing a log message on the status of the migration. | 100
+| pdrMigrationParams | Object | Options for the PDRs data migration | `{}`
+| pdrMigrationParams.parallelScanSegments | number | The number of [parallel scan] segments to use for PDRs data migration. The higher this number, the less time it will take to migrate all of your data, but also the more load that will be put on your PostgreSQL database. | 20
+| pdrMigrationParams.parallelScanLimit | number | The maximum number of records to return per each [parallel scan] of a segment. This option was mostly provided for testing and it is not recommended to set a value. | none
+| pdrMigrationParams.writeConcurrency | number | The maximum number of PDR records to write concurrently to PostgreSQL. The higher this number, the less time it will take to migrate all of your data, but also the more load that will be put on your PostgreSQL database. | 10
+| pdrMigrationParams.loggingInterval | number | How many records to migrate before printing a log message on the status of the migration. | 100
```

Comment:
Are all of these technically optional since they all have defaults?

Reply:
Yes, they are all optional